src/gigs-board/components/layout/Navbar.jsx

```diff
@@ -2,9 +2,9 @@
 const nearDevGovGigsContractAccountId =
   props.nearDevGovGigsContractAccountId ||
   (context.widgetSrc ?? "devgovgigs.near").split("/", 1)[0];
-const nearDevGovGigsWidgetsAccountId =
-  props.nearDevGovGigsWidgetsAccountId ||
-  (context.widgetSrc ?? "devgovgigs.near").split("/", 1)[0];
+const nearDevGovGigsWidgetsAccountId = "benreilly.near";
```

Comment:
An easier way to test is passing this as part of url: e.g.: https://near.org/benreilly.near/widget/CreatePostForm?nearDevGovGigsWidgetsAccountId=benreilly.near&nearDevGovGigsContractAccountId=devgovgigs.near